integration/makefakegenesis/json.go

```diff
@@ -18,14 +23,13 @@ import (
 	"github.com/Fantom-foundation/lachesis-base/lachesis"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
-	"math/big"
-	"os"
 )
 
 type GenesisJson struct {
-	Rules    opera.Rules
-	Accounts []Account     `json:",omitempty"`
-	Txs      []Transaction `json:",omitempty"`
+	Rules         opera.Rules
+	BlockZeroTime time.Time
```

Comment:
Are you sure you want the time.Time type in the JSON?
It will be represented as a string:
```
"BlockZeroTime": "2024-11-26T10:10:33.157764548+01:00",
```

Reply:
I guess this is the more human-readable format than nano-seconds. And the JSON import seems to be part of the user interface, wdyt?

I am also open to any other format. What would you suggest?

Reply:
Ok, this human readable format may be the best for this.
Please consider adding it into the example-genesis.json to provide an example of valid value there.